charts/rancher-istio/1.1.5/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: rancher-istio
-version: 1.1.1
-appVersion: 1.1.1
+version: 1.1.5
+appVersion: 1.1.5
```

Comment:
Should we distinguish the chart version from appVersion? It would be trouble if we need to release a new chart without an appVersion change.